rust/src/conf.rs

```diff
@@ -14,36 +14,40 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+use crate::model::common::ClientType;
+use std::time::Duration;
+
 #[derive(Debug, Clone)]
 pub struct ClientOption {
-    name_space: String,
-    access_url: String,
-    enable_tls: bool,
+    pub(crate) client_type: ClientType,
+    pub(crate) group: String,
+    pub(crate) namespace: String,
+    pub(crate) access_url: String,
+    pub(crate) enable_tls: bool,
+    pub(crate) timeout: Duration,
+    pub(crate) long_polling_timeout: Duration,
 }
 
 impl Default for ClientOption {
     fn default() -> Self {
         ClientOption {
-            name_space: "".to_string(),
+            client_type: ClientType::Producer,
+            group: "".to_string(),
+            namespace: "".to_string(),
```

Comment:
We currently have not exposed the concept of namespace to the client side, is it possible to remove the namespace here?

Reply:
Producer group makes no sense in 5.x series?